src/modules/validateInput.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const validateInput = (value) => {
+  if (isNaN(value)) {
+    console.warn('All characters must be digits');
+
+    return false;
+  }
+
+  if (value.length !== 4) {
+    console.warn('There should be 4 digits');
+
+    return false;
+  }
+
+  const userNumber = new Set(value);
```

Comment:
The value here should be converted to String before applying the Set function. This is because Set function applied directly to number will consider it as single entity and won't check for distinct digits.